templates/project-man/ingressroute.yaml

```
@@ -34,14 +34,14 @@ spec:
   entryPoints:
     - websecure
   routes:
-    - match: {{ include "jet-helm-chart.ruleHosts" . }} && {{include "jet-helm-chart.rulePathPrefixs" (dict "jetPrefix" .Values.ingressroute.subpath "componentPrefix" "/jet") }}
+    - match: {{ include "jet-helm-chart.ruleHosts" . }} && {{ printf "PathPrefix(`%s`)" (include "jet-helm-chart.buildPath" (dict "prefix" .Values.ingressroute.subpath "path" "/jet")) }}
```

这个是不是应该是 project_man 呀？
```suggestion
    - match: {{ include "jet-helm-chart.ruleHosts" . }} && {{ printf "PathPrefix(`%s`)" (include "jet-helm-chart.buildPath" (dict "prefix" .Values.ingressroute.subpath "path" "/project_man")) }}
```